src/test/java/com/google/gcp_variant_transforms/library/SchemaGeneratorImplTest.java

```diff
@@ -0,0 +1,247 @@
+  }
+
+  @Test
+  //column: names
+  public void testGetFields_whenGetNamesField_thenIsEqualTo() {
+    ImmutableList<TableFieldSchema> fields = schemaGen.getFields(vcfHeader);
+    UnmodifiableListIterator<TableFieldSchema> fieldIterator =
+        fields.listIterator(SchemaUtils.FieldIndex.NAMES);
+    TableFieldSchema namesField = fieldIterator.next();
+
+    assertThat(namesField.getName())
+        .isEqualTo(SchemaUtils.FieldName.NAMES);
+    assertThat(namesField.getDescription())
+        .isEqualTo(SchemaUtils.FieldDescription.NAMES);
+    assertThat(namesField.getMode()).isEqualTo(SchemaUtils.FieldMode.REPEATED);
+    assertThat(namesField.getType()).isEqualTo(SchemaUtils.FieldType.STRING);
+  }
+
+  @Test
+  //column: quality
+  public void testGetFields_whenGetQualityField_thenIsEqualTo() {
+    ImmutableList<TableFieldSchema> fields = schemaGen.getFields(vcfHeader);
+    UnmodifiableListIterator<TableFieldSchema> fieldIterator =
+        fields.listIterator(SchemaUtils.FieldIndex.QUALITY);
+    TableFieldSchema qualityField = fieldIterator.next();
+
+    assertThat(qualityField.getName())
+        .isEqualTo(SchemaUtils.FieldName.QUALITY);
+    assertThat(qualityField.getDescription())
+        .isEqualTo(SchemaUtils.FieldDescription.QUALITY);
+    assertThat(qualityField.getMode()).isEqualTo(SchemaUtils.FieldMode.NULLABLE);
+    assertThat(qualityField.getType()).isEqualTo(SchemaUtils.FieldType.FLOAT);
+  }
+
+  @Test
+  //column: filter
+  public void testGetFields_whenGetFilterField_thenIsEqualTo() {
+    ImmutableList<TableFieldSchema> fields = schemaGen.getFields(vcfHeader);
+    UnmodifiableListIterator<TableFieldSchema> fieldIterator =
+        fields.listIterator(SchemaUtils.FieldIndex.FILTER);
+    TableFieldSchema filterField = fieldIterator.next();
+
+    assertThat(filterField.getName())
+        .isEqualTo(SchemaUtils.FieldName.FILTER);
+    assertThat(filterField.getDescription())
+        .isEqualTo(SchemaUtils.FieldDescription.FILTER);
+    assertThat(filterField.getMode()).isEqualTo(SchemaUtils.FieldMode.REPEATED);
+    assertThat(filterField.getType()).isEqualTo(SchemaUtils.FieldType.STRING);
+  }
+
+  @Test
+  //column: calls record
+  public void testGetFields_whenGetCallsField_thenIsEqualTo() {
+    ImmutableList<TableFieldSchema> fields = schemaGen.getFields(vcfHeader);
+    UnmodifiableListIterator<TableFieldSchema> fieldIterator =
+        fields.listIterator(SchemaUtils.FieldIndex.CALLS);
+    TableFieldSchema callsField = fieldIterator.next();
+
+    assertThat(callsField.getName())
+        .isEqualTo(SchemaUtils.FieldName.CALLS);
+    assertThat(callsField.getDescription())
+        .isEqualTo(SchemaUtils.FieldDescription.CALLS);
+    assertThat(callsField.getMode()).isEqualTo(SchemaUtils.FieldMode.REPEATED);
+    assertThat(callsField.getType()).isEqualTo(SchemaUtils.FieldType.RECORD);
+  }
+
+  @Test
+  //column: calls record's subfields
```

Comment:
First comment should leave one space after "//", the following comments should do the same

Reply:
Thanks for all of the helpful comments, Zhiqiang!